docs/UserGuide.md

```diff
@@ -117,24 +135,84 @@ Examples:
 
 ### Listing assignments : `list`
 
-Format: `list [NUMBER]`
+Format: `list [NUMBER_OF_DAYS]`
 
-- Shows a list of assignments in your schedule within next `NUMBER` 
-days, starting from the current date (and current time).
-- A day represents 24 hours. 
-- `list` without `NUMBER` displays your entire list of assignments 
-stored in ProductiveNUS.
+You can list your assignments altogether with just typing `list`. Alternatively, you can type `list` followed by an index `NUMBER_OF_DAYS` to list your assignments with deadlines that are within the current date (and time) and `NUMBER_OF_DAYS` later.
 
+You can use `NUMBER_OF_DAYS` index to quickly view assignments that you need to complete soon! (with deadlines nearing).
+For example, `list 2` will show you assignments that are due 2 days (48 hours) from the current date (and current time).
+{: .alert .alert-gitlab-purple}
 
 Examples: 
-- `list 2` lists all your assignments with deadline 2 days (48 hours) from the current date (and current time). 
-- `list 3` lists all your assignments with deadline 3 days (72 hours) from the current date (and current time). 
-- `list ` lists all your assignments.
+- `list` lists all your assignments.
+- `list 3` lists all your assignments with deadline 3 days (72 hours) from the current date (and current time). For example, If the current date and time is 24/10/2020 12:00 pm, all assignments due from this date and time to 27/10/2020 12:00PM will be displayed.
+ 
+
+<div class="panel panel-info">
+**Pointers to note:**
+{: .panel-heading}
+<div class="panel-body">
+
+* `NUMBER_OF_DAYS` **must be a positive integer** 1, 2, 3, …​
+* **Only one** number can be keyed in. For example, `list 1 2` will not work. 
+
+</div>
+</div>
 
 ### Finding assignments : `find`
 
+You can find your assignments based on keywords (and prefixes) you enter. The types of keywords are the name, module code, deadline and priority level of assignments. You can find assignments with multiple keywords of the same type.
+
+Here is the table of prefixes that is used:
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;}
+.tg td{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg th{border-color:black;border-style:solid;border-width:1px;font-family:Arial, sans-serif;font-size:14px;
+  font-weight:normal;overflow:hidden;padding:10px 5px;word-break:normal;}
+.tg .tg-0lax{text-align:left;vertical-align:top}
+</style>
+<table class="tg">
+<thead>
+  <tr>
+    <th class="tg-0lax">Prefix</th>
+    <th class="tg-0lax">Type of keyword</th>
+    <th class="tg-0lax">Syntax with examples</th>
+    <th class="tg-0lax">Remarks</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
```

Comment:
For this table, I don't think the "type of keyword" column is very meaningful as the syntax column already mentions the type of keyword in all caps. Also, personally, I feel that the "syntax with example" column is a little cluttered. Maybe separating that column into 2 columns would be better? (one syntax column and one examples column)